ui/page/transaction/transactions_page.go

```diff
@@ -54,27 +51,43 @@ type TransactionsPage struct {
 
 	selectedTabIndex int
 
-	txTypeDropDown   *cryptomaterial.DropDown
+	txTypeDropDown *cryptomaterial.DropDown
+	walletDropDown *cryptomaterial.DropDown
+
 	transactionList  *cryptomaterial.ClickableList
 	previousTxFilter int32
 	scroll           *components.Scroll[*sharedW.Transaction]
 
-	tabs *cryptomaterial.ClickableList
+	tab *cryptomaterial.SegmentedControl
```

Comment:
Just `tab` is not a good name. What tab is this? `txCategoryTab`? Please rename.